README.md

```diff
@@ -88,6 +98,20 @@ Literal =
   | FLOAT_NUMBER
   | "true"
   | "false"
+ImportItem ::= ImportTree ";"
+ImportTree ::=
+    ImportTreeList
+  | Path
+  | Alias
+ImportTreeList ::= "{" ImportTreeListElements "}"
+ImportTreeListElements ::=
+    ImportTree
+  | ImportTree "," ImportTreeListElements
+Path ::=
+    PathSegment
+  | PathSegment "::" Path
+PathSegment ::= NameRef
+Alias ::= ImportTree "as" NameRef
```

Comment:
`Alias` is recursive on itself through `ImportTree` which makes `import ::foo as bar as foo` valid syntax.